assets/src/models/vehicle.ts

```diff
@@ -18,6 +18,9 @@ export const isGhost = (
 export const isLateVehicleIndicator = ({ id }: Ghost): boolean =>
   id.startsWith("ghost-incoming-")
 
+export const isLoggedOut = ({ operatorLogonTime, runId }: Vehicle) =>
+  operatorLogonTime === null && runId === null
```

Comment:
question (non-blocking): would it be worthwhile at this point to introduce a LoggedOutVehicle type with this guard? Or does that fit more into https://app.asana.com/0/1152340551558956/1204891002317847/f?

Reply:
I think given this is already a medium-sized PR it would be better to hold off and wait for that other ticket.